src/java.desktop/share/classes/java/awt/GridLayout.java

```diff
@@ -32,25 +32,36 @@
  * lays out a container's components in a rectangular grid.
  * The container is divided into equal-sized rectangles,
  * and one component is placed in each rectangle.
- * For example, the following is an applet that lays out six buttons
+ * For example, the following is a window that lays out six buttons
  * into three rows and two columns:
  *
- * <hr><blockquote>
+ * <hr>
+ * {@snippet lang='java':
  * <pre>
```

Comment:
```suggestion
 * {@snippet lang='java':
```
`<pre>` is inside the sample code.